### PR TITLE
update slackbuild link

### PR DESCRIPTION
**Pull request by spaette**

### CODE REVIEW

File: source/download.html.md [hunk 1]
```diff
@@ -22,8 +22,8 @@ like to have a binary hosted, please
...
 |[Gentoo](http://packages.gentoo.org/package/app-office/ledger) (3.2.1)                                                                                                                ||[`fink`](http://www.finkproject.org/) `install` [`ledger`](http://pdb.finkproject.org/pdb/package.php/ledger) (3.1)                                 |[NetBSD](https://pkgsrc.se/finance/ledger) package (3.2.1) |
 |[Arch Linux](https://www.archlinux.org/packages/community/x86_64/ledger/) (3.2.1)                                                                                                                      ||[`port`](https://www.macports.org/) `install` [`ledger`](https://trac.macports.org/browser/trunk/dports/finance/ledger/Portfile) (3.1.1)||||
-|[Fedora](https://src.fedoraproject.org/rpms/ledger/) (3.1.3)||||
-|[Slackware](http://slackbuilds.org/repository/14.2/business/ledger/) (3.1.3)||||
+|[Fedora](https://src.fedoraproject.org/rpms/ledger/) (3.2.1)||||
+|[Slackware](https://slackbuilds.org/repository/15.0/business/ledger/) (3.1.3)||||
```
Comment (afh) [inline source/download.html.md:26]: Should the version number also be changed? Checking the URL `3.2.1` is listed…
```suggestion
|[Slackware](https://slackbuilds.org/repository/15.0/business/ledger/) (3.2.1)||||
```

**Review by afh — COMMENTED** [review]
Thanks for the update, @spaette.

See my question below regarding the changes

**Review by tbm — APPROVED** [review]
Thank you!

I'll update the Slackware version as suggested by @afh.

**Comment by spaette** [comment]
@tbm 

thanks for fixing my mistake

the RedHat link which I checked does list 3.2.1 so that does not need to be reverted

**Pull request merged**